From c71099b0aa5fc413f778d82a8ea6df2cb745c968 Mon Sep 17 00:00:00 2001
From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Tue, 27 Sep 2022 23:39:54 +0200
Subject: Enable HEVC by default

Add a guard to avoid setting to be disabled upstream.

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 media/base/media_switches.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/media/base/media_switches.cc b/media/base/media_switches.cc
index e27c55be5d5db..d38d37a86f4e0 100644
--- a/media/base/media_switches.cc
+++ b/media/base/media_switches.cc
@@ -1227,7 +1227,7 @@ BASE_FEATURE(kPlatformEncryptedDolbyVision,
 
 #if BUILDFLAG(ENABLE_PLATFORM_HEVC)
 // Enables HEVC hardware accelerated decoding.
-BASE_FEATURE(kPlatformHEVCDecoderSupport, base::FEATURE_ENABLED_BY_DEFAULT);
+BASE_FEATURE(kPlatformHEVCDecoderSupport, base::FEATURE_ENABLED_BY_DEFAULT); // must be enabled
 
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_APPLE) || BUILDFLAG(IS_ANDROID)
 // Enables HEVC hardware accelerated encoding for Windows, Apple, and Android.
-- 
2.55.0

